app/controllers/games_controller.rb

```diff
@@ -0,0 +1,28 @@
+class GamesController < ApplicationController
+	# before_action :authenticate_user!
+  
+  def show
+  	@game = Game.find(params[:id])
+  end
+
+  def new
+  	@game = Game.new
+  end
+
+  def create
+  	@game = Game.new(game_params)
+  	if @game.save
+  		redirect_to game_path(@game)
+  	else
+  		render :new, :status => :unprocessable_entity 
+  	end
+
+  end
+
+  private
+
+  def game_params
+		params.require(:game).permit(:name)  	
+  end
+
+end
```

Comment:
I also have use spaces instead of tab:
```

    // The number of spaces a tab is considered equal to
    "tab_size": 2,

    // Set to true to insert spaces when tab is pressed
    "translate_tabs_to_spaces": true
```